debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/MongoDbSinkConnectorTask.java

```diff
@@ -51,7 +51,7 @@ public void start(final Map<String, String> props) {
         try {
             this.connectionContext = new MongoDbConnectionContext(config);
```

Comment:
Hey @rk3rn3r, unrelated to these changes but more for my own edification, is there a long-term goal for the `MongoDbConnectionContext` to be a member variable rather than a local variable?  It doesn't seem to be used anywhere else, so I wasn't sure if this could just be allocated and  used here locally?